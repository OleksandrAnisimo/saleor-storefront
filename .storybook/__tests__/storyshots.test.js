import initStoryshots from "@storybook/addon-storyshots";
import { multiSnapshotWithOptions } from "@storybook/addon-storyshots/dist/test-bodies";
import styleSheetSerializer from "jest-styled-components/src/styleSheetSerializer";
import { addSerializer } from "jest-specific-snapshot";

addSerializer(styleSheetSerializer);

initStoryshots({
  framework: "react",
  test: multiSnapshotWithOptions({})
});
