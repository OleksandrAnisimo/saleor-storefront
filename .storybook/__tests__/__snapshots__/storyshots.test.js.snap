// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots @components/atoms/Button Primary 1`] = `
<div
  className="sc-bdVaJa fkRMdE"
>
  <button
    className="sc-iwsKbI ddOSpf"
    color="primary"
    size="md"
  >
    <span
      className="sc-gqjmRU iGzszp"
      size="md"
    >
      Primary Button
    </span>
  </button>
  <div
    className="sc-bwzfXH cyotrw"
  />
</div>
`;

exports[`Storyshots @components/atoms/Button Secondary 1`] = `
<div
  className="sc-bdVaJa fkRMdE"
>
  <button
    className="sc-iwsKbI sc-gZMcBi cbmDhR"
    color="secondary"
    size="md"
  >
    <span
      className="sc-gqjmRU iGzszp"
      size="md"
    >
      Secondary Button
    </span>
  </button>
  <div
    className="sc-bwzfXH cyotrw"
  />
</div>
`;

exports[`Storyshots @components/atoms/Button Size Sm 1`] = `
<div
  className="sc-bdVaJa fkRMdE"
>
  <button
    className="sc-iwsKbI inyexi"
    color="primary"
    size="sm"
  >
    <span
      className="sc-gqjmRU dzPnpq"
      size="sm"
    >
      Small Button
    </span>
  </button>
  <div
    className="sc-bwzfXH cyotrw"
  />
</div>
`;
